### ISSUE TITLE
Checkbox - High Contrast, Checkbox selection is Not visible till mouse hover on it

### ISSUE THREAD

**Issue by MdAmeerMind** [microsoft/fluentui#12201]
Actual Behavior:
 In High contrast mode Black/white, checkbox select or not while clicking on it using mouse are visible till user move out mouse from checkbox

Repro: [https://developer.microsoft.com/en-us/fabric#/controls/web/checkbox](https://developer.microsoft.com/en-us/fabric#/controls/web/checkbox)

Repro Step:
1) Start High contrast mode(black/white).
2) Open Edge browser with above link.
3) Go to checkbox, lick on it using mouse.
Now observe that it's difficult to know that checkbox is selected or not till mouse move some where else on the screen.

**Comment by aneeshack4** [comment]
Thanks for reporting this @MdAmeerMind. I’ll work on getting a fix for this.

**Comment by aneeshack4** [comment]
To add some context: 
![image](https://user-images.githubusercontent.com/4161791/76041525-2fd32d00-5f07-11ea-9a39-9f196d5934a6.png)
Basically, this is due to on hover-focus checked state, the box is black. On rest checked state, it's blue so that's when it's clear that it's been checked not while active. I think we need to use a different semantic slot for the hover-focus state here: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts#L103 What do you think @cliffkoh?

**Comment by aneeshack4** [comment]
Just to clarify, @MdAmeerMind this is what you're talking about right? when you first click on the checkbox, the check mark appears but the box itself doesn't change color to purple (in high contrast white) and blue (in high contrast black) until you mouse away - is that the problem you're referring to? Because otherwise, the checked state is clear in both high contrast white and black: 
![image](https://user-images.githubusercontent.com/4161791/76444426-f4ff4800-6391-11ea-8356-22231950ded1.png)
![image](https://user-images.githubusercontent.com/4161791/76444433-f7fa3880-6391-11ea-9edd-71446fde1d94.png)

**Comment by MdAmeerMind** [comment]
Hi @aneeshack4 , Yes that is problem I am referring.

**Comment by aneeshack4** [comment]
@MdAmeerMind Apologies for the late response. Like I said in the previous comment, I think that a different semantic slot needs to be used for the hover-focus state. Tagged component owner @cliffkoh but he's currently out of office and most likely we need design opinion here so perhaps @betrue-final-final knows which slot should be used here.

**Comment by jasonjgeiger** [comment]
@aneeshack4, do you have status on when this issue might be resolved?

**Comment by SrikanthBoyapati** [comment]
@aneeshack4, can we get an update on this please?

**Comment by petewu100** [comment]
I don't repro the problem on the https://developer.microsoft.com/en-us/fluentui#/controls/web/checkbox test page with Fabric 6.164.8 in Chromium Edge 84.0.508.0.  Has the problem been fixed?

Note: this is also covered in https://github.com/microsoft/fluentui/issues/12299 and https://github.com/microsoft/fluentui/issues/12264 which are dup'd to each other.

**Comment by MdAmeerMind** [comment]
Hi @petewu100 , thank you for your response. Other bugs #12299 and #12264 are fixed.
But bug I have mention are still reproducible. validate on Chromium Edge 84.0.508.0.
Please refer https://github.com/microsoft/fluentui/issues/12201#issuecomment-597758735 this is issue.

Repro step:

1) High contrast mode
2) Click on checkbox, now observe that user won't understood whether checkbox is checked/ unchecked until mouse hover out from that checkbox.

**Comment by v-sagsah** [comment]
@petewu100 could you please provide an update on these ?

**Comment by xugao** [comment]
@betrue-final-final - I think we should just simplify the design by removing any on-hover color changing effect and only let the pointer cursor to indicate on hover. thoughts?
